packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -6,6 +6,16 @@ html {
   --cxl-wrap-width: 72rem;
   --cxl-wrap-padding: var(--lumo-space-m);
 
+  /**
+   * Vaadin Icons do not have "safe area" around the icon
+   * like in Lumo Icons which have 4px around.
+   *
+   * Use this value when both are next to each other and need same size.
+   *
+   * @see https://vaadin.com/docs/latest/components/icons
+   */
+  --cxl-vaadin-icon-padding: 4px;
```

Comment:
Doesn't this have to be dynamic based on `var(--vaadin-icon-size-...)` value?

I would maybe avoid going into a global solution rabbit hole in this PR.

Reply:
I've tried to make it dynamic with calculations you suggested but `--vaadin-icon-size-*` values were either less or more then 4px (eg. 3.75px). Docs specifically read "4px safe area" and I decided to use this value.

I tested various font sizes and icon always looked good.

Reply:
Aha, good point about the docs.

Let's only be more clear about the concept with `--cxl-vaadin-icon-safe-area` name then.